doc/dev_guide/internals/replication/orphan.rst

```diff
@@ -53,16 +50,13 @@ The following situations are possible.
 Here ``box.cfg{}`` is being called for the first time.
 A replica is joining but no replica set exists yet.
 
-    1.  Set status to 'orphan'.
-    2.  Try to connect to all nodes from ``box.cfg.replication``,
-        or to the number of nodes required by
-        :ref:`replication_connect_quorum <cfg_replication-replication_connect_quorum>`.
+    1.  Set the status to 'orphan'.
+    2.  Try to connect to all nodes from ``box.cfg.replication``.
         Retrying up to 3 times in 30 seconds is possible because this is bootstrap,
         :ref:`replication_connect_timeout <cfg_replication-replication_connect_timeout>`
         is overridden.
 
-    3.  Abort and throw an error if not connected to all nodes in ``box.cfg.replication`` or
-        :ref:`replication_connect_quorum <cfg_replication-replication_connect_quorum>`.
+    3.  Abort and throw an error if not connected to all nodes in ``box.cfg.replication``.
```

Comment:
The error will be thrown only if the node fails to connect to a majority of nodes from `box.cfg.replication`.
Just like if `replication_connect_quorum` was set to `#box.cfg.replication / 2 + 1`.